documentation/troubleshooting.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Troubleshooting Kubernetes vSphere Cloud Provider
+---
+This document is aimed to help debug issues in the Kuberentes vSphere Cloud Provider.
+
+For general Kubernetes Issue, Please visit [https://kubernetes.io/docs/tasks/debug-application-cluster/debug-cluster/](https://kubernetes.io/docs/tasks/debug-application-cluster/debug-cluster/)
+
+## What logs can I inspect?
+For digging deeper and help gather data for the technical support we need to see the log files.
+
+For vSphere Cloud Provider most often we need to look into Controller-Manager, Kubelet and API-server logs.
```

Comment:
We can format this info better. Let's chat offline.

Reply:
As discussed provided sections and added proper indentation.